platform-modules/actors/src/main/java/org/ekstep/learning/framework/FrameworkStore.java

```diff
@@ -0,0 +1,106 @@
+package org.ekstep.learning.framework;
```

Comment:
This class is almost same as **CollectionStore** class. Why don't you reuse the same Class.

Reply:
Deleted FrameworkStore and Renamed CollectionStore to HierarchyStore and used for framework.